Refactor modules (#2217)
pyansys/pyaedt: doc/source/API/Application.rst

PyAEDT apps
===========

> The PyAEDT API includes classes for apps and modules. You must initialize the 

Reviewer: ```suggestion
The PyAEDT API includes classes for apps and modules. You must initialize Ansys 
```

> app to get access to all modules and methods.

Reviewer: ```suggestion
Electronics Desktop (AEDT) to get access to all PyAEDT modules and methods.
```

.. image:: ../Resources/aedt_2.png
  :width: 800
  :alt: Ansys Electronics Desktop (AEDT) is a platform that enables true electronics system design.

> Available apps are:

Reviewer: ```suggestion
Available PyAEDT apps are:
```

.. autosummary::
   :toctree: _autosummary

   pyaedt.desktop.Desktop
   pyaedt.hfss.Hfss
   pyaedt.q3d.Q3d
   pyaedt.q3d.Q2d
   pyaedt.maxwell.Maxwell2d
   pyaedt.maxwell.Maxwell3d
   pyaedt.icepak.Icepak
   pyaedt.hfss3dlayout.Hfss3dLayout
   pyaedt.mechanical.Mechanical
   pyaedt.rmxprt.Rmxprt
   pyaedt.circuit.Circuit
   pyaedt.maxwellcircuit.MaxwellCircuit
   pyaedt.emit.Emit
   pyaedt.twinbuilder.TwinBuilder

All other classes and methods are inherited into the app class.

> The desktop app is implicitly launched in any of the other applications.

Reviewer: ```suggestion
AEDT is implicitly launched in any of the other apps.
```

> Before accessing an AEDT app the Desktop has to be launched and initialized.

Reviewer: ```suggestion
Before accessing an AEDT app the desktop app has to be launched and initialized.
```

> Desktop can be explicitly or implicitly initialized as in following examples.

Reviewer: ```suggestion
AEDT can be explicitly or implicitly initialized as in following examples.
```
